components/StoryCards/StoryCard.module.scss

```diff
@@ -161,20 +162,6 @@ $small-card-image-size: 230px;
     }
 }
 
-.medium,
-.big {
```

Comment:
> I guess it makes more sense to follow vertical and horizontal variants for story cards instead of big | medium | small sizes. However, that refactor may change lots of stuff but that's something we can consider to simplify CSS.

Yes, it would be really nice! Can we do this in scope of this PR?

Reply:
As discussed, let's do this in a separate PR.